framework/include/mesh/MooseMesh.h

```diff
@@ -486,7 +486,22 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
   /**
    * Set uniform refinement level
    */
-  void setUniformRefineLevel(unsigned int);
+  void setUniformRefineLevel(unsigned int, bool deletion = true);
+
+  /**
+   * Return a flag indicating whether or not we should skip remote deletion
+   * and repartition after uniform refinements. If the flag is true, uniform
+   * refinements will run more efficiently, but at the same time, there might
+   * be extra ghosting elements. The number of layers of additional ghosting
+   * elements depends on the number of uniform refinement levels.  This flag
+   * should be used only when you have a "fine enough" coarse mesh and want
+   * to refine the mesh by a couple of levels (less than five levels). Otherwise,
```

Comment:
How did you arrive at the number 5?

Reply:
Haha, good question. I magically figured that out.

OK, suppose you have one layer of geometric ghosting elements. After 5 levels of refinement, you would end up with 32  (2^5) layers of geometric ghosting elements. That is already pretty large for me.  For 3D problems, the refined mesh will be 32768 (8^5) times larger than the original one. 

Normally, I do only one or two levels of refinements.  

Should I rephrase the statement as "possibly less than five levels"?  Or should I say "less than 2"?

Reply:
How about
```suggestion
   * to refine the mesh by a few levels. Otherwise,
```
And then similarly replace elsewhere? It's not a big deal

Reply:
OK, I will do that

Reply:
That's an improvement.